addons/sounds/functions/fnc_removeSound.sqf

```diff
@@ -16,7 +16,11 @@ if (isNull _unit) exitWith {};
 
 // find object in array
 private _rmIndex = GVAR(soundList) findIf { (_x select 0) == _unit};
-GVAR(soundList) deleteAt _rmIndex;
+while{ _rmIndex != -1 } do {
+	GVAR(soundList) deleteAt _rmIndex;
+	_rmIndex = GVAR(soundList) findIf { (_x select 0) == _unit};
+};
+
```

Comment:
From 2.18 an onwards you can do it with single command if you got all the indexes. But this works for now. 
![billede](https://github.com/Crowdedlight/Crows-Electronic-Warfare/assets/7889925/2c56122c-75c4-4b8c-9e5b-85600e91ce80)